galgebra/printer.py

```diff
@@ -819,11 +819,6 @@ def tex(paper=(14, 11), debug=False, prog=False, pt='10pt'):
                     latex_line = '\\begin{equation*} ' + latex_line + ' \\end{equation*}\n'
 
         else:
-            latex_line = latex_line.replace(r'\left.', r'@@')  # Disabiguate '.' in '\left.'
-            latex_line = latex_line.replace(r'\right.', r'##')  # Disabiguate '.' in '\right.'
-            latex_line = latex_line.replace('.', r' \cdot ')  # For components of metric tensor
-            latex_line = latex_line.replace(r'@@', r'\left.')  # Restore '\left.'
-            latex_line = latex_line.replace(r'##', r'\right.')  # Restore '\right.'
```

Comment:
Although I do agree to remove these erroneous bits but it's a breaking change so I marked this PR as "need changelog"

Reply:
Agreed :)